pegasus/sites.v3/code.org/public/page_mode.haml

```diff
@@ -1,3 +0,0 @@
-page mode:
-= PageMode.get(request)
-
```

Comment:
this looks like a file that engineers use to quickly test what the page mode is.  @breville , maybe you can confirm whether it is needed?

Reply:
I don't think it's been used for a long time, so it's alright to remove.

Reply:
Sounds good, will resolve this convo.